### PR TITLE
Setup Eslint for SSO

### LINKED ISSUE #3155: Configure ESLint for `sso`
`src/api/sso` needs to be configured to add the new `@senecacdot/eslint-config-telescope` to `devDependencies` and configure `ESLint` to actually work.

### PR DESCRIPTION
**Pull request by JiaHua-Zou**
Fixes #3155 
## Issue This PR Addresses
Fix the `Eslint` configuration for `SSO` files.

## Type of Change

- [x] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Steps to test the PR
pnpm install
pnpm lint

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -17139,7 +17145,7 @@ packages:
     dev: true
 
   /strip-json-comments/2.0.1:
-    resolution: {integrity: sha1-PFMZQukIwml8DsNEhYwobHygpgo=, registry: https://registry.npmjs.com/, tarball: https://registry.npmjs.com/strip-json-comments/-/strip-json-comments-2.0.1.tgz}
```
Comment (aserputov) [inline pnpm-lock.yaml:17142]: I'm not sure why this changed.

File: src/api/sso/package.json [hunk 2]
```diff
@@ -5,7 +5,10 @@
   "description": "An SSO Authentication and Authorization Service",
   "scripts": {
     "dev": "env-cmd -f env.local nodemon src/server.js",
-    "start": "node src/server.js"
+    "start": "node src/server.js",
+    "eslint": "TIMING=1 eslint --config .eslintrc.js \"./src/**/*.js\" \"./test/**/*.js\"",
+    "eslint-fix": "eslint --config .eslintrc.js \"./src/**/*.js\" \"./test/**/*.js\" --fix",
```
Comment (aserputov) [inline src/api/sso/package.json:10]: what about the current directory? I'm not sure about this one too, I need @DukeManh

File: src/api/sso/package.json [hunk 3]
```diff
@@ -29,7 +32,10 @@
   },
   "devDependencies": {
     "@jcoreio/wait-for-postgres": "2.0.0",
+    "@senecacdot/eslint-config-telescope": "1.1.0",
     "env-cmd": "10.1.0",
-    "nodemon": "2.0.15"
+    "eslint": "7.32.0",
+    "nodemon": "2.0.15",
+    "supertest": "6.1.6"
```
Comment (tcvan0707) [inline src/api/sso/package.json:39]: We can remove `supertest` here, SSO doesn't use it I believe.

**Review by tcvan0707 — CHANGES_REQUESTED** [review]

File: src/api/sso/package.json (hunk 3 above)
Comment (JiaHua-Zou) [reply src/api/sso/package.json:39]: supertest is used in the `/api/sso/test/e2e/routes.test.js`

[the remark this answers is not in the document]
Comment (JiaHua-Zou) [reply src/api/sso/package.json:39]: supertest is used in the `/api/sso/test/e2e/routes.test.js`

File: pnpm-lock.yaml (hunk 1 above)
Comment (JiaHua-Zou) [reply pnpm-lock.yaml:17142]: I think this was changed when I isntall the eslint and @senecacdot/eslint-config-telescope

Comment (JiaHua-Zou) [reply pnpm-lock.yaml:17142]: I think this was changed when I isntall the eslint and @senecacdot/eslint-config-telescope

**Review by tcvan0707 — APPROVED** [review]

File: src/api/sso/package.json (hunk 2 above)
Comment (DukeManh) [reply src/api/sso/package.json:10]: Let's include every `js` file.
```suggestion
    "eslint-fix": "eslint --config .eslintrc.js \"./**/*.js\" --fix",
```

File: src/api/sso/package.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (JiaHua-Zou) [reply src/api/sso/package.json:39]: Should I add an eslint ignore on when we request `supertest` or add the `supertest` to the `package.json` becuase eslint doesn't like `supertest` isn't listed in the dependencies.

**Review by tcvan0707 — APPROVED** [review]

**Review by cindyledev — APPROVED** [review]

**Pull request merged**